ibis/pyspark/compiler.py

```diff
@@ -332,18 +332,18 @@ def compile_aggregator(t, expr, scope, fn, context=None, **kwargs):
         src_col = F.when(condition, src_col)
 
     col = fn(src_col)
-    if context is None:
+    if context == AggregationContext.ENTIRE:
```

Comment:
This is just for readability

Making this change here is kind of orthogonal but I use the same logic in the compilation function for `ReductionVectorizedUDF`